pkg/cloud/services/ec2/ami_test.go

```diff
@@ -184,10 +185,16 @@ func TestAMIs(t *testing.T) {
 		t.Run(tc.name, func(t *testing.T) {
 			g := NewWithT(t)
 
+			scheme, err := setupScheme()
+			g.Expect(err).NotTo(HaveOccurred())
+			client := fake.NewClientBuilder().WithScheme(scheme).Build()
```

Comment:
Can we extract a func out of these lines as same is used in line 453-456

Reply:
Actually I have already refactored it using `setupScheme()` method and I have checked other test files also and kept it like this to maintain the consistency.